ProcessLib/HeatTransportBHE/HeatTransportBHEProcessData.h

```diff
@@ -82,6 +89,12 @@ struct HeatTransportBHEProcessData
     std::unordered_map<int, int> _map_materialID_to_BHE_ID;
 
     std::vector<BHE::BHETypes> _vec_BHE_property;
+
+    // if bhe network exists python boundary condition
+    bool if_bhe_network_exist_python_bc;
+
+    //! Python object computing BC values.
+    BHEInflowPythonBoundaryConditionPythonSideInterface* py_bc_object;
```

Comment:
No plain pointers, please. This is a memory leak.

Reply:
fixed in commit #b23114c6